driver/src/models/order.rs

```diff
@@ -1,10 +1,15 @@
 use serde::Deserialize;
 use web3::types::{H160, U256};
 
+// TODO: refactor the struct to be an enum that can be either the stableX
+// version (slot index) or the Snapp version (slot_index + slot).
 #[derive(Debug, Clone, Deserialize, Eq, Ord, PartialEq, PartialOrd)]
 #[serde(rename_all = "camelCase")]
 pub struct BatchInformation {
+    // only applies to the Snapp version
     pub slot: U256,
+    /// the order index of a StableX order (the n-th order a user has ever
+    /// placed in the system)
     pub slot_index: u16,
```

Comment:
This should probably be called `order_id` then right? Just curious, we can change it in a separate PR (or get rid of the `BatchInformation` struct altogether).

Reply:
I ... am not sure. I took the comment from an explanation Felix gave me on slack. With the removal of the snapp as you say we can probably remove BatchInformation.

Reply:
Yes, we should refactor it into a single order_id now (but not in this PR).